baybe/constraints/continuous.py

```diff
@@ -31,3 +36,87 @@ class ContinuousLinearInequalityConstraint(ContinuousConstraint):
     The class has no real content as it only serves the purpose of
     distinguishing the constraints.
     """
+
+
+@define
+class ContinuousCardinalityConstraint(ContinuousConstraint):
+    """Class for continuous cardinality constraints.
+
+    Places a constraint on the set of nonzero (i.e. "active") values among the
+    specified parameter, bounding it between the two given integers,
+        ``min_cardinality`` <= | {1(p_i != 0)}_i | <= ``max_cardinality``
```

Comment:
dont really gethow nthis equation `{1(p_i != 0)}_i` is meant, can you elaborate?
Kronecker delta is typically done with two indices `1_{i,j}` (discrete version) or with minus `1(x-special_value)` (continuous) but I dont see that here so I dont quite get it

Reply:
Fixed using set notation with condition